modules/vaos/app/serializers/ccra/referral_detail_serializer.rb

```diff
@@ -38,7 +39,8 @@ class ReferralDetailSerializer
         }
       end
 
-      provider_info
+      # Transform keys to camelCase
```

Comment:
Note this is needed because nested objects do not get transformed by the `set_key_transform :camel_lower`